### ISSUE TITLE
[Question] Is it possible to have multiple Y axes for different sets of data?

### ISSUE THREAD

**Issue by iamrecursion** [pyrochlore/obsidian-tracker#121]
I'm using a particular instance of tracker as a visual way to spot correlations. However, not all of my measurements use the same scale for their values. I was wondering if there's a way by which you can have independent Y-axis scales for individual data sources (in my case `dvField`s) on the same tracker chart.

**Comment by Cityjohn** [comment]
Yes ofcourse, if you look in the example Vault in the first example bloodpressure you will find exactly that.

**Comment by iamrecursion** [comment]
Thank you for the pointer.
